src/components/MovieDescription/MovieDescription.js

```diff
@@ -30,6 +32,25 @@ const MovieDescription = ({info}) => {
     )
 }
 
+export default MovieDescription;
 
+MovieDescription.propTypes = {
+    info: PropTypes.shape({
+        tagline: PropTypes.string.isRequired,
+        overview: PropTypes.string.isRequired,
+        genres: PropTypes.arrayOf(PropTypes.string),
+        budget: PropTypes.number.isRequired,
+        revenue: PropTypes.number.isRequired,
+        runtime: PropTypes.number.isRequired
+    })
+}
 
-export default MovieDescription;
\ No newline at end of file
+MovieDescription.defaultProps = {
+    info: {
+        tagline: 'The world\'s best tagline',
+        overview: 'Trust me bro, just go and see it!',
+        budget: '--',
```

Comment:
I like that we can add a budget default value. I HATE that even if it comes in as a falsy value that we still have to conditionally render it. This isn't actually doing anything unless there are some objects without that key (which is possible)